source/documentation/deploying_services/sqs.md

```diff
@@ -0,0 +1,342 @@
+# Amazon SQS
+
+Amazon SQS provides "standard" (at-least-once delivery, best-effort ordering) and first-in-first-out queues.
+
+To provide any feedback on this feature, contact the GOV.UK PaaS team at
+[gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
+
+<h2 id="set-up-the-service">Set up the service</h2>
+
+### Provision an Amazon Web Service SQS queue
+
+Creating a service instance provisions an Amazon Web Service (AWS) Simple Queue
+Service (SQS) queue.
+
+The name of the service overall is `aws-sqs-queue`. There are two types of plan
+available for the service named `standard` and `fifo`.
+
+- `standard` plans configure queues that provide at-least-once delivery,
```

Comment:
Bullets should follow on from the lead-in sentence, so we should probably not use bullets here

Reply:
I have kept the bullets but tweaked this paragraph so that it better conforms to the standard

Reply:
yes, this is better